sdk/servicebus/azure-servicebus/setup.py

```diff
@@ -82,7 +82,8 @@
         'msrestazure>=0.4.32,<2.0.0',
         'azure-common~=1.1',
         'msrest>=0.6.17,<2.0.0',
-        'azure-core<2.0.0,>=1.6.0'
+        'azure-core<2.0.0,>=1.6.0',
+        "isodate>=0.6.0",
```

Comment:
do we want it to be "< 1.0.0" as well? if the dependency in autorest is just >=0.6.0, then I probably think it's for now. But still it would be better to check with anna or laurent on this.

Reply:
Confirmed with @lmazuel that >=0.6.0 is not a bad form. It's the same as msrest.

Reply:
Added an issue #12704 to track replacing isodate